unconvert.go

```diff
@@ -200,21 +201,14 @@ func main() {
 		}
 		wg.Wait()
 	} else {
-		var files []string
-		for f := range m {
-			files = append(files, f)
-		}
-		sort.Strings(files)
-		found := false
-		for _, f := range files {
-			if len(m[f]) != 0 {
-				found = true
+		var edits positionSlice
+		for _, positions := range m {
+			for pos := range positions {
+				edits = append(edits, pos)
 			}
-			print(f, m[f])
-		}
-		if found {
-			os.Exit(1)
```

Comment:
It exits with a non-zero status code, which is a common convention for UNIX tools like this.